pipelines/pingcap/tiflash/latest/pull_integration_next_gen.groovy

```diff
@@ -0,0 +1,505 @@
+                            pwd && ls -alh
+                            """
+                        }
+                        sh label: "clean unnecessary dirs", script: """
+                        git status
+                        git show --oneline -s
+                        rm -rf .git
+                        rm -rf contrib
+                        du -sh ./
+                        ls -alh
+                        """
+                    }
+                }
+            }
+        }
+
+        stage('Integration Tests') {
+            matrix {
+                axes {
+                    axis {
+                        name 'TEST_PATH'
+                        values 'tidb-ci', 'delta-merge-test', 'fullstack-test', 'fullstack-test2'
+                    }
+                }
+                agent{
+                    kubernetes {
+                        namespace K8S_NAMESPACE
+                        yamlFile POD_INTEGRATIONTEST_TEMPLATE_FILE
+                        defaultContainer 'docker'
+                        retries 5
+                        customWorkspace "/home/jenkins/agent/workspace/tiflash-integration-test"
+                    }
+                }
+                stages {
+                    stage("Test") {
+                        steps {
+                            dir("${WORKSPACE}/tiflash") {
+                                cache(path: "./", includes: '**/*', key: prow.getCacheKey('tiflash', REFS, 'it-build')){
+                                    println "restore from cache key: ${prow.getCacheKey('tiflash', REFS, 'it-build')}"
+                                    sh label: "debug info", script: """
+                                    printenv
+                                    pwd && ls -alh
+                                    """
+                                    dir("tests/${TEST_PATH}") {
+                                        echo "path: ${pwd()}"
+                                        sh label: "debug docker info", script: """
+                                        docker ps -a && docker version
+                                        """
+                                        script {
+                                            def pdBranch = component.computeBranchFromPR('pd', REFS.base_ref, REFS.pulls[0].title, 'master')
+                                            // def tikvBranch = component.computeBranchFromPR('tikv', REFS.base_ref, REFS.pulls[0].title, 'master')
+                                            def tikvBranch = "dedicated-next-gen_linux_amd64"
+                                            def tidbBranch = component.computeBranchFromPR('tidb', REFS.base_ref, REFS.pulls[0].title, 'master')
+
+                                            sh label: "run integration tests", script: """
+                                            PD_BRANCH=${pdBranch} TIKV_BRANCH=${tikvBranch} TIDB_BRANCH=${tidbBranch} TAG=${tiflash_commit_hash} BRANCH=${REFS.base_ref} ./run.sh
```

Comment:
```suggestion
                                            PD_BRANCH=${pdBranch} TIKV_BRANCH=${tikvBranch} TIDB_BRANCH=${tidbBranch} TAG=${tiflash_commit_hash} BRANCH=${REFS.base_ref} ENABLE_NEXT_GEN=true ./run.sh
```
Add a env so that I can change the deployment for next-gen of running integration test in the tiflash repo